analyzers/tests/SonarAnalyzer.UnitTest/Rules/Hotspots/PermissiveCorsTest.cs

```diff
@@ -47,10 +47,10 @@ public class PermissiveCorsTest
                 .Verify();
```

Comment:
Clean as we code, `PermissiveCors_CS` should be named `PermissiveCors_CSharp9` to match the language version.